packages/compass-schema/src/components/coordinates-minichart/utils.js

```diff
@@ -12,12 +12,21 @@ export async function _getHereTileBoxes() {
   const rawTileBoxes = await fetch(COPYRIGHT_URL).then((response) =>
     response.json()
   );
-  return rawTileBoxes.normal.map((attr) => ({
-    ...attr,
-    boxes: attr.boxes.map((box) =>
-      L.latLngBounds(L.latLng(box[0], box[1]), L.latLng(box[2], box[3]))
-    ),
-  }));
+  const result = rawTileBoxes.copyrights.in
```

Comment:
Actually one thing I haven't noticed at first, seems like base light.day / light.night also mentions jp and dtm copyrights, we should probably add them here
```suggestion
  const result = [...rawTileBoxes.copyrights.in, ...rawTileBoxes.copyrights.jp, ...rawTileBoxes.copyrights.dtm]
```

Reply:
> A style may use more than one data source. In that case, all the copyright blocks defined by .resources.base.styles.$STYLE_NAME must be used